packages/typo3-docs-theme/assets/sass/layout/navigation/_main_navigation.scss

```diff
@@ -37,6 +37,10 @@
         padding: calc(#{$spacer} / 4) 0;
         padding-right: 1.5em;
         text-decoration: none;
+        &:focus-visible {
+            color: $white;
```

Comment:
Why using a background-color and not just the common outline?

Additionally, the current solution looks not really good:
![mainnav](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/76806d29-268e-440b-8ab2-33ee708ffd28)

- text should have a padding left and right
- The + symbol is not really visible anymore

Reply:
as I already wrote in the other PR, because I was not able to do it better. The plus does not work with the keyboard anyway. The menu needs a makeover which someone :tm: has to do.